<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommitTaskDialog</class>
 <widget class="QDialog" name="CommitTaskDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TaskUpload</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>file</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>size</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>progress</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>speed</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>describe</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="quitPushButton">
       <property name="text">
        <string>quit</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addUploadPushButton">
       <property name="text">
        <string>addUpload</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="clearPushButton">
         <property name="text">
          <string>clearCompleted</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
